### DISCUSSION TITLE
An example of a JUnit5 test of the picocli String Boot starter

### DISCUSSION THREAD

**Discussion by roded** [remkop/picocli#2253] [Q&A] [answered]
Hi!
Any chance anyone has a simple example of a JUnit5 test for the picocli Spring Boot starter?
I've not been able to get it working with `spring.main.web-application-type=none`.
Thanks!

**Comment by remkop** [comment, marked as answer]
Is this what you are looking for?

https://github.com/remkop/picocli/blob/main/picocli-spring-boot-starter/src/test/java/picocli/spring/boot/autoconfigure/example/test/ExampleTest.java

https://github.com/remkop/picocli/blob/main/picocli-spring-boot-starter/src/test/java/picocli/spring/PicocliSpringFactoryTest.java

**Reply by roded** [reply]
Thanks @remkop, that helps.
